joi-validations/chat.validation.js

```diff
@@ -1,11 +1,14 @@
-const {uuid} = require('./general.validations');
+const {uuid, array} = require('./general.validations');
 
 const ChatValidation = {
   sendFollowSystemMessage: {
     body: {
       channel_id: uuid.required(),
       target_follow_user_id: uuid.required()
     }
+  },
+  findOrCreateChannelBySignedSender: {
+    members: array(uuid).min(1)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The validation for `findOrCreateChannelBySignedSender` only checks if the `members` array has at least one element. However, it does not validate if the elements in the array are unique. This could lead to unexpected behavior if duplicate UUIDs are passed. Consider adding a uniqueness check to the validation.
```diff
-    findOrCreateChannelBySignedSender: {
-      members: array(uuid).min(1)
+    findOrCreateChannelBySignedSender: {
+      members: array(uuid).min(1).unique()
    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->